xil_carma_cloud/cdasim_config/.env

```diff
@@ -3,3 +3,7 @@ DOCKER_ORG=usdotfhwastoldev
 TIME_SYNC_TOPIC=time_sync
 INFRASTRUCTURE_ID=rsu_1234
 SIMULATION_MODE=TRUE
+#ROS Middleware implementation for SIM PCs
+RMW_IMPLEMENTATION=rmw_fastrtps_cpp
```

Comment:
If we just include `RMW_IMPLEMENTATION_OVERWRITE` here, we don't have to include 
```
    environment:
      RMW_IMPLEMENTATION_OVERWRITE: ${RMW_IMPLEMENTATION}
```
in the docker-compose.yml. Which might be easier cognitive complexity

Reply:
I don't think it works like that misheel. In other words I think the docker compose environment variables only work for replacing docker compose instructions, not for contain shell commands. So adding 
```docker-compose
service:
    name: carma

    command " export RMW_IMPLEMENTATION=${RMW_IMPLEMENTATION_OVERWRITE}"
```
 will only resolve if we set it in the container/servive.

Reply:
ah okay that makes sense